go.mod

```diff
@@ -216,3 +216,5 @@ require (
 	sigs.k8s.io/json v0.0.0-20221116044647-bc3834ca7abd // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.3.0 // indirect
 )
+
+replace github.com/authgear/oauthrelyingparty => github.com/tung2744/authgear-oauthrelyingparty v1.2.0-3684db06
```

Comment:
v1.3.0 was published. Please use it instead.

Reply:
Switched to v1.3.0, thanks!